src/controllers/users.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const { UserService } = require('../services/users.service');
+
+const getAllUsers = (req, res) => {
+  const userService = new UserService();
```

Comment:
You need to create a `UserSerivce` once.
Use a singleton pattern if you want to create it in the class

https://www.freecodecamp.org/news/singleton-design-pattern-with-javascript/

```suggestion
```